sli/tarrayobj.cc

```diff
@@ -582,7 +582,7 @@ bool TokenArrayObj::operator==( const TokenArrayObj& a ) const
   if ( size() != a.size() )
     return false;
 
-  Token *i = begin(), *j = a.begin();
+  Token* i = begin(), * j = a.begin();
   while ( i < end() )
     if ( !( *i++ == *j++ ) )
       return false;
```

Comment:
This file has nothing to do with the pull request and just got in due to clang reformatting. Please remove from PR.